src/sass/_variables.scss

```diff
@@ -12,3 +12,11 @@ $form-border-color: #292929;
 
 // section-blog-posts
 $post-info-text-color: #676767; // converted from rgb(103,103,103)
+
+//section--gallery
+$second-column-color: #e8edf7;;
+$first-column-color: #f4f5f7;
+$border: #e2e2e2;
```

Comment:
These names are too general. Look at the old code from this project. It's not `text-color`, for instance, but `post-info-text-color`. It inditicates – "what is it?" (color), "what we style?" (info-text), "where is it?" (post). `border` can be eveywhere, and we don't know if it's about border-color or maybe border-style as well.

Reply:
Fixed by add more specific variable names